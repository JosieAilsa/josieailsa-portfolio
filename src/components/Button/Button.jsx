import "./Button.scss"

const Button = ({content, classNames, callback}) => {
    return ( <>
    <button className={classNames} role = "button" onClick={callback}>{content}</button>
    </>);
}
 
export default Button;