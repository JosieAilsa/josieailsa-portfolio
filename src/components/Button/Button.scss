@use "../../assets/sass/abstracts/variables" as *;

.hero__button{
    background-color: $color-pink;
    border: 2px solid $color-green-opacity;
    padding: 1rem;
    align-self: center;
    font-size: 1rem;
    color: $color-green;
    font-family: $font-body;
    font-weight: 500;
    border-radius: 2rem;
    margin-top: 2rem;
}
.hero__button--active:hover{
        background-color: $color-green;
        color: $color-pink;
        transition: 0.25s ease-in-out;
}