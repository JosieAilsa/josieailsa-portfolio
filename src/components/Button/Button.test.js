import { customRender } from "../../utils/testutils";
import Button from "./Button";

test("Button should render", () => {
  const { container } = customRender(<Button />);
  expect(container).toMatchSnapshot();
});
