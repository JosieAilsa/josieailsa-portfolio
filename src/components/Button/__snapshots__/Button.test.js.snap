// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render 1`] = `
<div>
  <button
    role="button"
  />
</div>
`;
